v1/flutter/README.md

```diff
@@ -0,0 +1,1125 @@
+# Building a Flutter Application with Nitric
+
+In this guide we'll go over how to create a basic Flutter application using the Nitric framework as the backend. Dart does not have native support on AWS, GCP, or Azure, so by using the Nitric framework you can use your skills in Dart to create an API and interact with cloud services in an intuitive way.
```

Comment:
Maybe make the readme more like others in the examples and direct people to the docs guide via a link. So we can maintain in one spot.